src/pages/groupComparison/GroupComparisonPage.tsx

```diff
@@ -280,6 +303,18 @@ export default class GroupComparisonPage extends React.Component<IGroupCompariso
                         >
                             Save to new comparison session
                         </button>
+                        <button
+                            className="btn btn-xs btn-default"
+                            onClick={this.store.clearDragUidOrder}
+                        >
+                            Reset
+                        </button>
+                    </div>
+                    <div className="btn btn-xs btn-none"
+                         onClick={()=>{ this.unsavedOrderWarningDismissed = true; }}
```

Comment:
lets put this is action().   i know there is actually no real reason, but i want to keep the convention alive for other devs